container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -1285,7 +1289,6 @@ instance_groups:
           memory: 2800
           virtual-cpus: 4
           healthcheck:
-            readiness:
```

Comment:
rebase on develop so you don't need this

Reply:
Search this line `    properties.cflinuxfs2-rootfs.trusted_certs: '"((ROOTFS_TRUSTED_CERTS))((#ROOTFS_TRUSTED_CERTS))\n((/ROOTFS_TRUSTED_CERTS))((INTERNAL_CA_CERT))"'
` in this file (~2178). There is equivalent for cflinuxfs3 but there is one for sle and opensuse stacks. I think you need one for cflinuxfs3 as well.